### ISSUE TITLE
Replace bootstrap icons with heroicons

### ISSUE THREAD

**Issue by DavidAJohn** [DavidAJohn/FotoStorio#4]
There is a new bootstrap icon in use in the application - just the camera icon used as a logo in the header, the footer and on the login/register screens.

For the sake of consistency, it would probably be better to switch them over to heroicons instead. That will allow me to remove the inline styles I've had to use to position the existing icon.

There's also a perfectly good camera icon in the heroicons collection.

**Comment by DavidAJohn** [comment]
Ironically, the first heroicon that I've found that I don't really like is the camera icon. The bootstrap-icons camera is much better, I think, particularly at larger sizes.

My main concern here was getting rid of the inline styles I'd had to use to position the icon exactly where I wanted it, so rather than switch to the heroicon camera, I switched to using the .svg bootstrap-icons camera instead, which then allows me to remove the inline-styles and position the icon next to the text using Tailwind classes.

Apart from the crazy svg path info, the styling around the icon is now a lot simply, with no need for any padding or negative margins on the containing divs.
